owp-config.el

```diff
@@ -136,6 +136,24 @@ multi path."
         :sort-by :date
         :category-index t)))
 
+(defun owp/get-injected-template (key)
```

Comment:
It's not a good way of converting `(list :key1 "val1" :key2 "val2")` to hash map with keys `"key1"` and `"key2"`, but I don't know a better way.